amr-wind/boundary_conditions/wall_models/WallFunction.cpp

```diff
@@ -105,6 +124,9 @@ void VelWallFunc::wall_model(
             const auto& den = rho_lev.const_array(mfi);
             const auto& eta = eta_lev.const_array(mfi);
 
+	    const amrex::Real dx_dz = dx[0] / dx[2];
+            const int z_dx_integer = static_cast<int>(std::round(dx_dz));
```

Comment:
this makes me nervous. What are you trying to do here?

Reply:
Forgot to eliminate this.